examples/browser/dapp.html

```diff
@@ -4,15 +4,15 @@
 <head>
   <meta charset="utf-8">
   <title>QuarkChain DApp Example</title>
-  <script src="https://cdn.jsdelivr.net/npm/quarkchain-web3@2.0.0/dist/quarkchain-web3.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/quarkchain-web3@2.1.0/dist/quarkchain-web3.js"></script>
 </head>
 
 <body>
   <h1>Send Tx</h1>
   <form id="sendTxForm">
     <br> To:
     <br>
-    <input type="text" name="to" value="0x000000000000000000000000000000000000000000000001">
```

Comment:
why change this default value?

Reply:
It's because 'xhr2-cookies' library used by Web3.providers.HttpProvider can not work on browser~

Reply:
> why change this default value?

Changed back the default value